src/Devices/LiquidCrystal_TC.cpp

```diff
@@ -68,6 +68,7 @@ void LiquidCrystal_TC::writeLine(const char* text, uint16_t line) {
 void LiquidCrystal_TC::writeLine(const __FlashStringHelper* text, uint16_t line) {
   // copy from program memory to SRAM and then call other function
   char buffer[17];
-  strcpy_P(buffer, (PGM_P)text);
+  size_t line_length;
```

Comment:
Is this variable used?

Reply:
No, it sure isn't, and I have no memory of that line of code. Thanks for catching that!